cmake/modules/AddSwift.cmake

```diff
@@ -820,6 +820,9 @@ function(_add_swift_library_single target name)
       if(NOT "${SWIFT_${SWIFTLIB_SINGLE_SDK}_ICU_I18N_INCLUDE}" STREQUAL "${SWIFT_${SWIFTLIB_SINGLE_SDK}_ICU_UC_INCLUDE}")
         include_directories("${SWIFT_${SWIFTLIB_SINGLE_SDK}_ICU_I18N_INCLUDE}")
       endif()
+
+      # We need to include cmark at this point (not in swift_common_standalone_build_config_cmark) using MSVC
```

Comment:
My understanding is that, at the point at which CMark was included before this change, there was no target that we were adding these flags. For other compilers, this is fine. However, CMake's MSVC generator understands this as a nop and doesn't do anything.